drivers/src/mailbox.rs

```diff
@@ -394,15 +431,60 @@ impl MailboxRecvTxn<'_> {
         Ok(())
     }
 
+    /// Sends `data.len()` bytes to the mailbox FIFO
+    /// Transitions from Execute --> RdyForData
     ///
-    /// Transitions from Execute --> Idle
+    /// # Arguments
     ///
-    pub fn complete(&mut self, success: bool) -> CaliptraResult<()> {
+    /// * `data` - data buffer.
+    ///
+    /// # Returns
+    ///
+    /// Status of Operation
+    ///
+    pub fn copy_response(&mut self, data: &[u8]) -> CaliptraResult<()> {
```

Comment:
Can we make this function private? Nobody should be calling it from the outside.

Reply:
This is a good point. It made sense for copy_request to allow the caller to separate the copy and the complete call. But I agree. I don't see a reason why the same option would be needed for the response.